test-suite/js/src/test/scala/org/scalajs/testsuite/niobuffer/SupportsTypedArrays.scala

```diff
@@ -7,12 +7,13 @@
 \*                                                                      */
 package org.scalajs.testsuite.niobuffer
 
-import org.junit.{Assume, BeforeClass}
+import org.junit.BeforeClass
+import org.junit.Assume._
 
 import org.scalajs.testsuite.utils.Platform
 
 trait SupportsTypedArrays {
   @BeforeClass def assumeThatContextSupportsTypedByteArrays(): Unit = {
-    Assume.assumeTrue(Platform.areTypedArraysSupported)
+    assumeTrue("Typed arrays are supported",  Platform.areTypedArraysSupported)
```

Comment:
Oh, there is an ambiguity on the interpretation of the message, it could be the description of the assumption that failed or the message of the exception that was thrown. I will rewrite it as `"Assumed typed arrays are supported"` to avoid this ambiguity. It will look like this:

```scala
[warn] Test assumption in test org.scalajs.testsuite.... failed: org.junit.AssumptionViolatedException: Assumed typed arrays are supported, took ...
```